staging/src/k8s.io/kube-scheduler/config/v1beta2/types_pluginargs.go

```diff
@@ -70,6 +70,10 @@ type NodeResourcesFitArgs struct {
 	// A resource group name can't contain '/'.
 	// +listType=atomic
 	IgnoredResourceGroups []string `json:"ignoredResourceGroups,omitempty"`
+
+	// ScoringStrategy selects the node resource scoring strategy.
+	// +listType=atomic
+	ScoringStrategy ScoringStrategy `json:"scoringStrategy,omitempty"`
```

Comment:
ah, there is a misunderstanding, this needs to be a reference, only the internal type should be non-reference

Reply:
this must be a reference.